README.md

```diff
@@ -1 +1,230 @@
-# terraform-aws-ec2-instance-group
\ No newline at end of file
+# terraform-aws-ec2-instance-group [![Build Status](https://travis-ci.org/cloudposse/terraform-aws-ec2-instance-group.svg?branch=master)](https://travis-ci.org/cloudposse/terraform-aws-ec2-instance-group)
+
+Terraform Module for providing N general purpose EC2 hosts.
+
+If you only need to provision a single EC2 instance, consider using the [terraform-aws-ec2-instance](https://github.com/cloudposse/terraform-aws-ec2-instance) module instead.
+
+**IMPORTANT** This module by-design does not provision an AutoScaling group. It was designed to provision a discrete number of instances suitable for running stateful services such as databases (E.g. Kafka, Redis, etc). 
+
+
+Included features:
+* Automatically create a Security Group
+* Option to switch EIP attachment
+* CloudWatch monitoring and automatic reboot if instance hangs
+* Assume Role capability
+
+## Usage
+
+Note: add `${var.ssh_key_pair}` private key to the `ssh agent`.
+
+Include this repository as a module in your existing terraform code.
+
+
+### Simple example:
+
+```hcl
+module "instance" {
+  source                      = "git::https://github.com/cloudposse/terraform-aws-ec2-instance-group.git?ref=master"
+  namespace                   = "${var.namespace}"
```

Comment:
let's use concrete `namespace`, `stage`, `name` and other attributes in `README` examples.
easier to read and provides a complete example